src/App.css

```diff
@@ -0,0 +1,132 @@
+/* Root variables */
```

Comment:
Don't you think this would look better with a gradient tho?

Reply:
Hahaha, of course I am going to use gradients you silly, later on